app/app.js

```diff
@@ -17,8 +20,20 @@ const startServer = () => {
   const app = express();
 
   app.use(express.json());
+  app.use(infoLogger);
   app.use('/api', usersRouter, suggestUsersRouter, groupsRouter, usersGroupRouter);
+  app.use(errorLogger);
   app.listen(process.env.PORT || 3000, () => console.log('Server is running'));
+
+  process.on('uncaughtException', err => {
```

Comment:
If this error happens, server also should return res with 500.

Reply:
As we discussed, i will leave only logging for this.